src/radical/pilot/agent/rm/base.py

```diff
@@ -18,7 +18,7 @@
 RM_NAME_TORQUE      = 'TORQUE'
 RM_NAME_YARN        = 'YARN'
 RM_NAME_SPARK       = 'SPARK'
-
+RM_NAME_DEBUG       = 'DEBUG'
```

Comment:
I used the DEBUG rm for, well, debugging - it can fake a summit like node layout on localhost.  Its not complete, but already useful...

Reply:
This is going to be extremely useful for integration tests now that I think about it! Do you think we can expand it for the rest of the RMs?

Reply:
Yes, we should be able to expand it to replace all RMs in a test setup! :-)